Blackjack/Deck.swift

```diff
@@ -10,14 +10,15 @@ import Foundation
 
 class Deck {
     var cards: [Card] = []
-    let titles = [Card.titles.jack, Card.titles.queen, Card.titles.king, Card.titles.ace]
-    
+    // Veli - I still dont think we should have titles as an enum in the Card class, if we do we need to include the numbers as well imo
+    let titles = ["two", "three", "four", "five", "six", "seven", "eight", "nine", "ten", Card.titles.jack.rawValue, Card.titles.queen.rawValue, Card.titles.king.rawValue, Card.titles.ace.rawValue]
+
     init() {
-        for j in 1...10 {
-            cards.append(Card(value: j, suit: Card.suit.spades.rawValue))
-            cards.append(Card(value: j, suit: Card.suit.clubs.rawValue))
-            cards.append(Card(value: j, suit: Card.suit.hearts.rawValue))
-            cards.append(Card(value: j, suit: Card.suit.diamonds.rawValue))
+        for j in 0...12 {
```

Comment:
Just noticed: you changed the loop indices so that you can index the titles array correctly but now the cards are getting assigned incorrect values

Reply:
hence i did the calcValue method